### PR TITLE
add colors, dark theme

### PR DESCRIPTION
**Pull request by iqmulator**

### CODE REVIEW

File: src/dotnet/Chat.UI.Blazor/Components/LeftPanel/LeftPanelButtons.razor [hunk 1]
```diff
@@ -5,7 +5,37 @@
     <div class="c-spacer"></div>
     <div class="c-content">
         <div class="c-logo" @onclick="@OnLogoClick">
-            <img draggable="false" class="w-8 h-8 fill-current" alt="Actual Chat" src="/favicon.svg"/>
+            <svg class="w-8 h-8" viewBox="0 0 37 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (alexis-kochetov) [inline src/dotnet/Chat.UI.Blazor/Components/LeftPanel/LeftPanelButtons.razor]: Please don't use SVG markup at Razor - move to the file for caching purposes

File: src/dotnet/Chat.UI.Blazor/Components/Settings/SettingsModal.razor [hunk 2]
```diff
@@ -13,7 +13,7 @@
     var showNotification = !isGuestOrNone && _enableIncompleteUI;
     var showLanguages = true;
     var showUserInterface = true;
-    var showThemeToggle = _enableIncompleteUI;
+    var showThemeToggle = account.IsAdmin;
```
Comment (alexis-kochetov) [inline src/dotnet/Chat.UI.Blazor/Components/Settings/SettingsModal.razor]: please use IncompleteUI feature toggle

**Review by alexis-kochetov — CHANGES_REQUESTED** [review]

File: src/dotnet/Chat.UI.Blazor/Components/LeftPanel/LeftPanelButtons.razor [hunk 3]
```diff
@@ -5,7 +5,7 @@
     <div class="c-spacer"></div>
     <div class="c-content">
         <div class="c-logo" @onclick="@OnLogoClick">
-            <img draggable="false" class="w-8 h-8 fill-current" alt="Actual Chat" src="/favicon.svg"/>
+            <logo-button-svg />
```
Comment (crui3er) [inline src/dotnet/Chat.UI.Blazor/Components/LeftPanel/LeftPanelButtons.razor:8]: I think it's useless to create web-components for just displaying SVG image. Just use img tag for that purpose.

File: src/dotnet/Chat.UI.Blazor/Components/LeftPanel/logo-button-svg.lit.ts [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import {customElement, property} from "lit/decorators.js";
+import {css, html, LitElement} from "lit";
+
+@customElement('logo-button-svg')
```
Comment (crui3er) [inline src/dotnet/Chat.UI.Blazor/Components/LeftPanel/logo-button-svg.lit.ts:4]: I think it's useless to create web-components for just displaying SVG image. Just use img tag for that purpose.

**Review by crui3er — CHANGES_REQUESTED** [review]

File: src/dotnet/Chat.UI.Blazor/Components/LeftPanel/LeftPanelButtons.razor (hunk 1 above)
Comment (crui3er) [reply src/dotnet/Chat.UI.Blazor/Components/LeftPanel/LeftPanelButtons.razor]: Don't make lit component. Use just img tag with svg file.

**Pull request merged**